.github/workflows/pull_request.yaml

```diff
@@ -38,18 +57,47 @@ jobs:
             fi
           done
         shell: bash
+      - name: Travis to GH actions
+        run: |
+          find ./ -type f -exec sed -i -e 's/TRAVIS_BUILD_DIR/GITHUB_WORKSPACE/g' {} \;
+          find ./ -type f -exec sed -i -e 's/TRAVIS_EVENT_TYPE/GITHUB_EVENT_NAME/g' {} \;
+          find ./ -type f -exec sed -i -e 's/TRAVIS_BRANCH/\{GITHUB_REF##*\/\}/g' {} \;
+          find ./ -type f -exec sed -i -e 's/TRAVIS_PULL_REQUEST_BRANCH/\{GITHUB_REF##*\/\}/g' {} \;
+          find ./ -type f -exec sed -i -e 's/TRAVIS_TAG/\{GITHUB_REF##*\/\}/g' {} \;
+          find ./ -type f -exec sed -i -e 's/-z "$TRAVIS_TAG" //g' {} \;
+          find ./ -type f -exec sed -i -e 's/${TRAVIS_PYTHON_VERSION}/3.7/g' {} \;
+          find ./ -type f -exec sed -i -e 's/"$TRAVIS_PULL_REQUEST" != "false"/"$GITHUB_EVENT_NAME" = "pull_request"/g' {} \;
+          find ./ -type f -exec sed -i -e 's/"$TRAVIS_PULL_REQUEST" = "false"/"$GITHUB_EVENT_NAME" != "pull_request"/g' {} \;
+          find ./ -type f -exec sed -i -e 's/travis_fold:start:/::group::/g' {} \;
+          find ./ -type f -exec sed -i -e 's/travis_fold:end:$containerlog/::endgroup::/g' {} \;
+          find ./ -type f -exec sed -i -e 's/.\/.travis\/check_commit.sh/#.\/.travis\/check_commit.sh/g' {} \;
+      - name: Before Install
+        run: |
+          .travis/before_install.sh
+          echo "::set-env name=TEST::${{ matrix.env.TEST }}"
+          cd ..
+          find ./ -type f -exec sed -i -e 's/timeout 90 bash/timeout 120 bash/g' {} \;
+          find ./ -type f -exec sed -i -e 's/\/usr\/local\/bin\/kubectl/$(which kubectl)/g' {} \;
+          find ./ -type f -exec sed -i -e 's/# containers are not running yet./sudo cp -i \/etc\/rancher\/k3s\/k3s.yaml $HOME\/.kube\/config/g' {} \;
```

Comment:
these changes were for `k3s`

Reply:
Yeah, originally I figured I'd purposely leave them there to make the delta against plugins smaller, but code clarity is important to, so I just removed them.